osCam/osCam/settings.py

```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-import environ
+# import environ
```

Comment:
did we mean to comment this out? There is more in this page I see as well (down where our Email notification settings live)